docs/plugins/first_plugin.md

```diff
@@ -206,8 +206,8 @@ Lastly, let's make a few small changes to `setup.cfg`.
    here.  Assume nothing about your user's environment!  Not even napari.)
 
 2. We need to instruct setuptools to *include* that `napari.yaml` file
-   when it bundles our package for distribution, by adding it to the
-   **`[options.package_data]`**
+   when it bundles our package for distribution, by adding
+   **`include_package_data = True`** to the `[options]` section.
```

Comment:
What do you think about linking out to the setuptools docs here?  They're linked to further up in the page, but it might be nice to something here.

```suggestion
2. We need to instruct [setuptools](https://setuptools.pypa.io/en/latest/userguide/datafiles.html) to *include* that `napari.yaml` file
   when it bundles our package for distribution, by adding
   **`include_package_data = True`** to the `[options]` section.
```